recipes/pynumero_build/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+
+#cd third_party/ASL
+#./getASL.sh
+#cd solvers
+#./configurehere
+
+#if [[ "$OSTYPE" == "linux-gnu" ]]; then
+#    find . -name "makefile" -exec sed -i "s/CFLAGS = -DNo_dtoa -fPIC -O/CFLAGS = -fPIC -O/g" {} \;
+#elif [[ "$OSTYPE" == "darwin"* ]]; then
+#    find . -name "makefile" -exec sed -ie 's/CFLAGS = -DNo_dtoa -fPIC -O/CFLAGS = -fPIC -O/g' {} \;
+#fi
+#make
+#cd ../../../
```

Comment:
Please remove the unnecessary commented code

Reply:
Done